.github/workflows/post-merge-tests.yml

```diff
@@ -0,0 +1,94 @@
+name: Post-merge APC tests
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
```

Comment:
With this, the workflow will run only if `rust` or `toml` files are touched, anything else which might impact running the guests?

Reply:
Sounds good